### ISSUE TITLE
CGI entry is not considered an "app"

### ISSUE THREAD

**Issue by ThiefMaster** [unbit/uwsgi#1189]
I have a case where I only use uWSGI to serve a legacy CGI script.

However, `cgi = ...` doesn't seem to be considered an "app" (no call to `uwsgi_add_app`). This results in `need-app` failing if set and also uWSGI switching to full dynamic mode. While I never used that mode I don't think it's something I want when using uWSGI just for CGI.

**Comment by xrmx** [comment]
Sorry but i can't see how you can expect to use cgi and need-app at the same time, in cgi you execute a script and you don't load an app at startup. Next time please use the mailing list to discuss this kind of issues. Thanks.

**Comment by ThiefMaster** [comment]
Sure, CGI scripts are not really an app - but does it make sense that I'm forced into dynamic mode, thus allowing clients with access to uWSGI (even though I kind of trust anything running on the machine) define new apps?!

**Comment by xrmx** [comment]
Yeah, that is definitely suboptimal to say the least. Have you checked that you can load a python application through nginx parameters?

**Comment by ThiefMaster** [comment]
no, haven't tested that since i'm not using any python stuff in that specific case (and i'm a lazy person ;))
